app/views/poligon-number.html

```diff
@@ -0,0 +1,12 @@
+<div class="widget col-md-12">
+  <span class="widgettitle">{{name}}</span>
+</div>
+<p class="widgetnumber">
+  <span>
+    <img class="poligonimage" ng-src="images/{{image}}">
+    <span ng-show="(value|exists) && value !== NaN">{{ value }}</span>
+    <span ng-hide="(value|exists) && value !== NaN" class="medium-character missing-data" translate="chart.metric.missing-data" style="position: relative;">NA</span >
```

Comment:
Do we need the `position` css rule? If so, maybe we can move it to one of the stylesheets?